src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -809,26 +992,30 @@ private Integer retrieveActiveConferenceCallStatistics(Map<String, String> stati
 
         statistics.put(ACTIVE_CONFERENCE_ID_LABEL, getJsonProperty(activeConference, "id", String.class));
         Long conferenceStartTimestamp = getJsonProperty(activeConference, "startTime", Long.class);
-        if(null != conferenceStartTimestamp) {
+        if (null != conferenceStartTimestamp) {
             statistics.put(ACTIVE_CONFERENCE_START_TIME_LABEL, String.valueOf(new Date(conferenceStartTimestamp)));
         }
 
         // Adding i+1 instead of i so terminals and connections are listed starting with 1, not 0
         if (terminals != null) {
-            for(int i = 0; i < terminals.size(); i++) {
+            for (int i = 0; i < terminals.size(); i++) {
                 int terminalNumber = i + 1;
                 statistics.put(String.format(ACTIVE_CONFERENCE_TERMINAL_ADDRESS_LABEL, terminalNumber), getJsonProperty(terminals.get(i), "address", String.class));
                 statistics.put(String.format(ACTIVE_CONFERENCE_TERMINAL_SYSTEM_LABEL, terminalNumber), getJsonProperty(terminals.get(i), "systemID", String.class));
             }
         }
         if (connections != null) {
-            for(int i = 0; i < connections.size(); i++) {
+            for (int i = 0; i < connections.size(); i++) {
                 int connectionNumber = i + 1;
                 statistics.put(String.format(ACTIVE_CONFERENCE_CONNECTION_TYPE_LABEL, connectionNumber), getJsonProperty(connections.get(i), "callType", String.class));
                 statistics.put(String.format(ACTIVE_CONFERENCE_CONNECTION_INFO_LABEL, connectionNumber), getJsonProperty(connections.get(i), "callInfo", String.class));
             }
         }
-        return getJsonProperty(activeConference, "id", Integer.class);

```

Comment:
sorry, could not comment on actual lines; Github does not give me the option.

Please extract `terminals.get(i)` into local object and reuse in lines 1003 and 1004.
Please extract `connections.get(i)` into local object and reuse in lines 1010 and 1011.